### PR TITLE
Add make request scene

### PR DESCRIPTION
**Pull request by bliu23**
The main components are all complete, has a date/time picker, inputs, etc.

TODO: 
1. We need to figure out best practices for react native styling including (but not limited to) where to place our StyleSheets + constants etc
2. MakeRequest scene is a parent component to many child components. Still need to pass a fn that alerts parent + sets state based on child input (this will be done in a separate PR, maybe when I have more time, or maybe when we're doing our actual POST requests)

### CODE REVIEW

File: client/js/MakeRequestScreen/DateTimePicker.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		//The same needs to be done for all input...		
+	}
+
+	render() {
+		/*
+			on some after confirm, we should pass in a function that is called that sets some state in the parent for form submission.
+		*/
+		return (
+			<View>
+				{<Text style={styles.inputHeader}>{this.props.type} Date</Text>}
```
Comment (bliu23) [inline client/js/MakeRequestScreen/DateTimePicker.js]: oops

File: client/js/MakeRequestScreen/DateTimePicker.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	_showDateTimePicker = (selected) => {
+		this.setState({ isDateTimePickerVisible: true });
+	}
+
+	_hideDateTimePicker = () => this.setState({ isDateTimePickerVisible: false });
+
+	_handleDatePicked = (date) => {
+		this.setState({time: date});
+		this._hideDateTimePicker();
+		// console.log(this.state.active + ' has been set to: ' + date);
```
Comment (bliu23) [inline client/js/MakeRequestScreen/DateTimePicker.js]: remove

File: client/js/MakeRequestScreen/DateTimePicker.js (hunk 1 above)
Comment (bliu23) [inline client/js/MakeRequestScreen/DateTimePicker.js]: brackets

File: client/js/MakeRequestScreen/MakeRequest.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+
+	render() {
+		function onPressHandle() {
+			console.log("Pressed!");
+		}
+		return (
+			<View style={styles.container}>
+				<View style={styles.inputView}>
+					<FormInput style={styles.singleLine} title={'Title'} placeholder={'Your short title'} />
```
Comment (bliu23) [inline client/js/MakeRequestScreen/MakeRequest.js:38]: singleLine multiline discrepancy

**Review by parkandrew — APPROVED** [review]

**Review by ArchKevin22 — APPROVED** [review]

File: client/js/MakeRequestScreen/MakeRequest.js (hunk 3 above)
Comment (bliu23) [reply client/js/MakeRequestScreen/MakeRequest.js:38]: going to keep as is since it makes sense to use camel case? maybe multiLine would be apt actually.

**Pull request merged**